src/app/interfaces/responses.interfaces.ts

```diff
@@ -0,0 +1,67 @@
+export interface loggedUserInfo {
```

Comment:
common pattern to call interfaces is to start from I - then we understand it is interface
please rename like `ILoggedUserInfo` etc